tests/e2e/fixtures/statsFixtures.js

```diff
@@ -263,6 +262,64 @@ const seasonTwoMatches = [
   },
 ];
 
+const seasonFourMatches = [
+  ////////////
+  // Week 1 //
+  ////////////
+  // Player 1 week 1
+  {
+    player1: 'player1',
+    player2: 'player2',
+    winner: 'player2',
+    startTime: dayjs().subtract(1, 'day').valueOf(),
+    endTime: dayjs().subtract(1, 'day').valueOf(),
```

Comment:
If the Current Season started 2 weeks ago, then week 1 of the current season is two weeks ago
```suggestion
    startTime: dayjs().subtract(2, 'week').valueOf(),
    endTime: dayjs().subtract(2, 'weel').valueOf(),
```